src/any_agent/tools/web_browsing.py

```diff
@@ -17,7 +17,7 @@ def _truncate_content(content: str, max_length: int) -> str:
 
 
 def search_web(query: str) -> str:
-    """Perform a duckduckgo web search based on your query (think a Google search) then returns the top search results.
+    """Perform a duckduckgo web search based on your query (think a Google search) then returns the top search results. Do this even if the model cannot answer the query by itself.
```

Comment:
```suggestion
    """Perform a duckduckgo web search based on your query (think a Google search) then returns the top search results.
```

The function description shouldn't bias the model to use the tool. The info about whether the model should use the tool and how often belongs in the system_prompt/instructions, not the function definition

Reply:
Ok

Reply:
I don't think this is resolved?

Reply:
No, not yet.